.lintr

```diff
@@ -0,0 +1,3 @@
+linters: linters_with_defaults(
+  line_length_linter(120),
+  cyclocomp_linter = NULL)
```

Comment:
We disable cyclocomp_linting for now. 
We may want to improve this in the future.